packages/plugins/persisted-queries/README.md

```diff
@@ -0,0 +1,63 @@
+## `@envelop/persisted-queries`
+
+TODO
+
+## Getting Started
+
+```
+yarn add @envelop/persisted-queries
```

Comment:
Isn't persisted-operations the better wording?

Reply:
Currently I am intentionally using this to differentiate form existing "persisted-operations" plugin.
This is currently a draft to discuss how to progress given overlap, if it encounters favour this might be merged within existing "persisted-operations"; it's something to discuss with project owners.

BTW, if you want my opinion, it's arguable.
You send a request to your server, and a single request can potentially contain multiple operations; yet it would have only a single hash/id.
In this case you're not persisting the operation (as you don't have an id per operation), but the whole query string body.
Hence, you are persisting the query string value, which content can involve multiple operations.

Also, persisted queries seems to be the de-facto convention, as documented in [Relay](https://relay.dev/docs/guides/persisted-queries/) and [Apollo](https://www.apollographql.com/docs/apollo-server/performance/apq/), just to mention some big players; but there are definitely more.

Reply:
Should we ride the edge?

Reply:
Sounds like I am backed, and so I would be happy to do so!
Will check with owners of the project and eventually change the name of this plugin to persisted-documents before fully opening the PR.

I think eventually, in case we decide to favour this over current "persisted-operations" plugin; the existing one could simply be deprecated and removed allowing this to become the standard.

Again, really looking forward to hear from The Guild on how to solve the overlap first.

Reply:
@santino @yaacovCR @n1ru4l Regarding the naming, we use the name "Persisted Operations" in Hive as of today. So, my view is that irrespective of what name we choose, we should try to stick to one name in both the plugin as well as hive to maintain consistency. Otherwise, both are explanatory according to me.

Reply:
My opinion is that persisted-documents is semantically correct and persisted-operations isn't, because as explained above you're not persisting operations but full GraphQL documents.
@yaacovCR seems to agree and has provided a good reference for glossary.

Having said this, as the owners of the project you are definitely free to make your decision, I just want to provide a recommendation.
I totally get your point that The Guild should use the same conventions across all their tools, it will surely be appreciated by your users :)

Reply:
That's a endless discussion :D 
We used the term `documents` in codegen, and got some user rejections because they were used to `queries` (which doesn't make sense)... 

Basically, for me, it's:
* `document` - a DocumentNode, that can hold any kind of AST.
* `operation` - Anything executable (query/mutation/subscription).

Reply:
And there is also Source, to throw more stuff into the mix.
Technically the string containing the Document, which includes the Operations, is a GraphQL Source.

Anyway, was a nice discussion and I care a lot about semantics, but I don't feel like insisting.
Persisted-operations be it, if you feel like so :)